docs/en/conf.md

```diff
@@ -0,0 +1,189 @@
+#--binlog_match_logoffset_interval=1000
+# Whether to notify the follower to synchronize immediately when data is written
+--binlog_notify_on_put=true
+# The maximum size of the binlog file, in M
+--binlog_single_file_max_size=2048
+# Master-slave synchronization batch size
+#--binlog_sync_batch_size=32
+# The interval between binlog sync and disk, in milliseconds
+--binlog_sync_to_disk_interval=5000
+# If there is no new data synchronization wait time, in milliseconds
+#--binlog_sync_wait_time=100
+# binlog filename length
+#--binlog_name_length=8
+# The interval for deleting binlog files, in milliseconds
+#--binlog_delete_interval=60000
+# Whether binlog enables crc verification
+#--binlog_enable_crc=false
+
+# Thread pool size for performing io-related operations
+#--io_pool_size=2
+# Execute the thread pool size for tasks such as deleting tables, sending snapshots, load snapshots, etc.
+#--task_pool_size=8
+# Configure the data directory, multiple disks are separated by English symbols
+--db_root_path=./db
+# Configure the data recycle bin directory, the data of the drop table will be placed here
+--recycle_bin_root_path=./recycle
+
+# snapshot conf
+# Configure the time to do snapshots, the time of day. For example, 23 means taking a snapshot at 23 o'clock every day.
+--make_snapshot_time=23
+# Check interval for snapshots
+#--make_snapshot_check_interval=600000
+# Set the offset threshold of the snapshot, if the offset difference from the last snapshot is less than this value, no new snapshot will be generated
+#--make_snapshot_threshold_offset=100000
+# snapshot thread pool size
+#--snapshot_pool_size=1
+# Whether snapshot compression is enabled. Can be set to off, zlib, snappy
+#--snapshot_compression=off
+
+# garbage collection conf
+# The time interval for performing expired deletion, in minutes
+--gc_interval=60
+# Thread pool size to perform expired deletion
+--gc_pool_size=2
+
+# send file conf
+# Maximum number of retries to send a file
+#--send_file_max_try=3
+# block size when sending files
+#--stream_block_size=1048576
+# Bandwidth limit when sending files, the default is 20M/s
+--stream_bandwidth_limit=20971520
+# Maximum number of retries for rpc requests
+#--request_max_retry=3
+# rpc timeout, in milliseconds
+#--request_timeout_ms=5000
+# If an exception occurs, the retry wait time, in milliseconds
+#--request_sleep_time=1000
+# Retry wait time for file sending failure, in milliseconds
+#--retry_send_file_wait_time_ms=3000
+#
+# table conf
+# The maximum height of the first level jump table
+#--skiplist_max_height=12
+# The maximum height of the second level jump table
+#--key_entry_max_height=8
+
+# loadtable
+# The number of data bars to submit a task to the thread pool when loading
+#--load_table_batch=30
+# Number of threads to load snapshot files
+#--load_table_thread_num=3
+# The maximum queue length of the load thread pool
+#--load_table_queue_size=1000
+```
+
+## apiserver configuration file conf/tablet.flags
```

Comment:
The Configuration File for APIServer: conf/tablet.flags

Reply:
Thank you!